### DISCUSSION TITLE
Errors object clearing out on submit

### DISCUSSION THREAD

**Discussion by Ekeu** [react-hook-form/react-hook-form#10071] [Q&A]
Hello,

I'm facing a situation where when I submit my form, if there is an error in the `errors` object, it clears out.

Below is a record of my screen

https://user-images.githubusercontent.com/23312714/223972235-8c69e7a5-b63a-466e-8106-690aad8554bb.mov

At first sight, I think it's because the query parameter changes, and react-hook-form thinks I'm on a new page. Hence, when I press the save button to submit, it triggers the validations that are on the new page; if there are no errors, it clears out the errors object.

**Comment by leapful** [comment]
@Ekeu 

It might be caused because unmounted form fields from your "Banner" tab section when switching to "Native Display" tab. Try to apply `display: none` for inactive tab content instead of destroy it when switching tab.
